tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_fillbug.py

```diff
@@ -0,0 +1,33 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall
+import time
+from gaiatest.apps.bzlite.app import BugzillaLiteStage
+
+
+class TestFillBug(GaiaTestCase):
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+        self.test_app = {
+            'name': 'Bugzilla Lite Stage',
+            'url': 'http://bzlite-staging.herokuapp.com/manifest.webapp'}
+        self.logger.info('Test data: %s' % self.test_app)
+        self.marionette.execute_script(
+            'navigator.mozApps.install("%s")' % self.test_app['url'])
+        confirm_install = ConfirmInstall(self.marionette)
+        confirm_install.tap_confirm()
+        self.push_resource('IMG_0001.jpg')
+
+    def test_fill_new_bug(self):
+
+        bug_title = "Automation Test "
+        bug_description = "Description of an Automation test"
+
+        bugzilla_lite = BugzillaLiteStage(self.marionette)
+        bugzilla_lite.launch()
+        bugzilla_lite.dismiss_tooltip()
+        bugzilla_lite.dismiss_content()
+        time.sleep(1)
```

Comment:
I added this sleep because after this dismiss it's refreshing the page so in some cases it put just the paswword and login, also sometimes blocked and sometimes works